### PR TITLE
Replace webpack with parcel

### PR DESCRIPTION
**Pull request by fregante**
I was trying to fix []() https://github.com/sindresorhus/notifier-for-github/issues/251 but of course webpack’s config was a PITA to update and conform to ESM, so the next logical step was… https://github.com/fregante/browser-extension-template/pull/44

To get started I ran:

```
npm i -D parcel@nightly @parcel/config-webextension@nightly
```

### CODE REVIEW

File: source/options.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import browser from 'webextension-polyfill';
```
Comment (fregante) [inline source/options.js:1]: No longer globally-available; This is required until https://github.com/fregante/browser-extension-template/issues/48 is resolved

File: source/options.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import 'webext-base-css';
```
Comment (fregante) [inline source/options.css:1]: Found the cleanest way to import this

File: .npmrc [hunk 3]
```diff
@@ -1 +0,0 @@
-package-lock=false
```
Comment (fregante) [inline .npmrc:1]: Lockfiles are useful for apps/extensions

File: package.json [hunk 4]
```diff
@@ -7,31 +7,32 @@
...
 		"esm": "^3.2.25",
 		"lodash.merge": "^4.6.2",
 		"moment": "^2.24.0",
 		"npm-run-all": "^4.1.5",
+		"parcel": "^2.0.0-nightly.634",
 		"sinon": "^7.3.2",
 		"sinon-chrome": "^3.0.1",
-		"size-plugin": "^1.2.0",
-		"stylelint": "^10.1.0",
+		"stylelint": "^13.12.0",
```
Comment (fregante) [inline package.json:34]: Update was required due to an `@import` bug

File: package.json [hunk 5]
```diff
@@ -51,17 +52,17 @@
...
+		}
 	},
 	"stylelint": {
 		"extends": "stylelint-config-xo",
 		"rules": {
 			"declaration-no-important": null
 		}
+	},
+	"webExt": {
+		"sourceDir": "distribution"
```
Comment (fregante) [inline package.json:74]: Extra: now you can run `web-ext run` in the root.

File: package.json [hunk 6]
```diff
@@ -7,31 +7,32 @@
...
+		"parcel": "^2.0.0-nightly.634",
 		"sinon": "^7.3.2",
 		"sinon-chrome": "^3.0.1",
-		"size-plugin": "^1.2.0",
-		"stylelint": "^10.1.0",
+		"stylelint": "^13.12.0",
 		"stylelint-config-xo": "^0.15.0",
-		"terser-webpack-plugin": "^1.4.1",
-		"webpack": "^4.38.0",
-		"webpack-cli": "^3.3.6",
```
Comment (fregante) [inline package.json:34]: Always a relief

**Comment by fregante** [comment]
The options page appears to work. Untested elsewhere because I'm notification zero 😌

**Comment by sindresorhus** [comment]
Thanks for doing this. Seems to be working fine.

**Comment by fregante** [comment]
🙌

**Pull request merged**